src/stores/spaces.ts

```diff
@@ -1,6 +1,8 @@
 import { defineStore } from 'pinia';
 import { currentNetwork } from '@/networks';
+import { useStorage } from '@vueuse/core';
```

Comment:
```suggestion
import { useStorage } from '@vueuse/core';
import { currentNetwork } from '@/networks';
```

nit. usually we put npm imports above internal ones.